modules/admin/proc-configuring-postgresql-with-tls-using-operator.adoc

```diff
@@ -0,0 +1,106 @@
+[id="proc-configuring-postgresql-with-tls-using-operator_{context}"]
+= Configuring an external PostgreSQL instance with TLS connection using the Operator
+
+You can edit the Operator default setting to configure an external PostgreSQL database server, such as the Crunchy database, with TLS connection. 
+
+.Prerequisites
+
+* You are using a supported version of PostgreSQL. For more information, see the link:https://access.redhat.com/support/policy/updates/developerhub[Product life cycle page].
+* You have the following details:
+** `db-host`: Denotes your PostgreSQL instance Domain Name System (DNS) or IP address 
+** `db-port`: Denotes your PostgreSQL instance port number, usually `5432`
+** `username`: Denotes the user name to connect to your PostgreSQL instance
+** `password`: Denotes the password to connect to your PostgreSQL instance
+* You have installed the {product-very-short} application by using the Operator or Helm Chart.
```

Comment:
The prerequisite shouldn't be to install the RHDH instance, because that would lead to a conflict. You want to have the operator installed, but not the RHDH itself.

```suggestion
* You have installed the Operator for {product-very-short}.
```